lib/support/cli.js

```diff
@@ -211,6 +211,13 @@ module.exports.parseCommandLine = function parseCommandLine() {
       type: 'boolean',
       group: 'Graphite'
     })
+    .option('graphite.use1_0', {
```

Comment:
Rather than specifying the release number, I'd name the option after the actual feature. Maybe graphite.arrayTags (defaulting to false to keep the current behaviour), alternatively graphite.stringTags (defaulting to true).